src/processor/InstanceProcessor.ts

```diff
@@ -110,7 +110,13 @@ export class InstanceProcessor {
     const flatInstanceArray = toPairs(getPathValuePairs(inputJSON));
     flatInstanceArray.forEach(([path], i) => {
       const assignmentRule = new ExportableAssignmentRule(path);
-      assignmentRule.value = getFSHValue(i, flatInstanceArray, instanceOfJSON.type, fisher);
+      assignmentRule.value = getFSHValue(
+        i,
+        flatInstanceArray,
+        instanceOfJSON.type,
+        target.id,
```

Comment:
I think you can technically have ids that are the same if the resource type is different. Do you think it is worth making the name that gets logged here be something like `{target.instanceOf} {target.id}` or `{target.id} of {target.instanceOf`?